dataset_configs/georgian/mcv/config.yaml

```diff
@@ -0,0 +1,121 @@
+documentation: |
+  MCV Georgian
+  ############
+  This config was originally designed for the Mozilla Common Voice (MCV)
+   `MCV <https://commonvoice.mozilla.org/>`__ dataset 17.0 release, 
+   but should work for any subsequent releases as well.
```

Comment:
Could you please delete the extra space in these 2 lines? So that the ` and b are underneath the T in "This". 
Currently the docs look a little bit off because of these extra spaces